doc/system_requirements.md

```diff
@@ -0,0 +1,181 @@
+# System Requirement Specification for Exasol Glue Connector
+
+## Introduction
+
+Exasol Glue Connector (EGC) is a connector for AWS Glue Studio. It provides an extended connector for AWS Glue Studio with Exasol capabilities to read data from and write data to an Exasol database.
+
+## About This Document
+
+### Target Audience
+
+The target audience are users, software engineers, data engineers and quality assurance engineers that build data pipelines and transform data using AWS Glue Studio. See section ["Stakeholders"](#stakeholders) for more details.
+
+### Goal
+
+The goal of EGC is to provide connector so that users can integrate Exasol database into their AWS Glue Studio workflows. It enables reading data from Exasol as a source and loading data from other sources supported by AWS Glue Studio into an Exasol database.
```

Comment:
```suggestion
The goal of EGC is to provide a connector so that users can integrate Exasol database into their AWS Glue Studio workflows. It enables reading data from Exasol as a source and loading data from other sources supported by AWS Glue Studio into an Exasol database.
```